messaging/src/test/java/org/axonframework/commandhandling/AsynchronousCommandBusTest.java

```diff
@@ -68,6 +68,7 @@ void setUp() throws Exception {
     void testDispatchWithCallback() throws Exception {
```

Comment:
I think there's merit in validating whether the span logic is correctly followed in this unit test class.